### DISCUSSION TITLE
FireMonaco - UserColor should not be a required field

### DISCUSSION THREAD

**Discussion by Blakeinstein** [0xTheProDev/Operational-Transformation#96] [Bug Report]
If not specified, a color should be picked at random. Firepad had this originally I believe.

**Comment by 0xTheProDev** [comment]
Hi @Blakeinstein,

Feel free to add your wrapper around `FireMonaco`, and I believe for most usecase API for top-level component will be widely opinionated. And hence kept API very minimal just as much needed to bind two adapters with Plain Text Editor.
